evap/staff/tools.py

```diff
@@ -292,3 +301,21 @@ def find_next_unreviewed_evaluation(semester, excluded):
         .filter(contributions__textanswer_set__state=TextAnswer.State.NOT_REVIEWED) \
         .annotate(num_unreviewed_textanswers=Count("contributions__textanswer_set")) \
         .order_by('vote_end_date', '-num_unreviewed_textanswers').first()
+
+
+def remove_user_from_represented_and_ccing_users(user, ignored_users=None, test_run=False):
+    remove_messages = []
+    ignored_users = list() if ignored_users is None else ignored_users
```

Comment:
you could write `ignored_users = ignored_users or []` instead

Reply:
or just use `(user, ignored_users=[], test_run=False)` (keep in mind that the actual list instance is then shared across calls, so you can't do that if you modify the list in the method. Shouldn't be a problem here, though)

Reply:
Using '[]' as default parameter throws an error in the codecov test because of the problem @He3lixxx mentioned (which wozld not be an actual problem im this case). I changed it because of that... The Suggestion of @janno42 seems the best solution then...